src/ext/framework/mod.rs

```diff
@@ -230,6 +257,15 @@ impl Framework {
                 });
 
                 if let Some(command) = self.commands.get(&built) {
+                    if message.is_private() {
+                        if command.guild_only {
+                            return
+                        }
+                    } else {
+                        if command.dm_only {
+                            return
+                        }
+                    }
```

Comment:
This can also be:

```rust
if (message.is_private() && command.guild_only) || command.dm_only {
    return;
}
```
It'll go over 80 chars, but so long as it's <= 100 it should be fine.